tests/dao/redis/test_metric_timeseries.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+from typing import Generator
+from typing import List
+
+import pytest
+
+from redisolar.dao.redis import MetricDaoRedisTimeseries
+from redisolar.models import MeterReading
+from redisolar.models import MetricUnit
+
+TESTING_SITE_ID = 1
+NOW = datetime.datetime.utcnow()
+
+
+@pytest.fixture
+def metric_dao(redis_timeseries, key_schema):
+    yield MetricDaoRedisTimeseries(redis_timeseries, key_schema)
+
+
+@pytest.fixture
+def readings(metric_dao) -> Generator[List[MeterReading], None, None]:
+    """Generate 72 hours worth of data."""
+    time = NOW - datetime.timedelta(hours=72)
+    readings = []
+    for i in range(72 * 60, -1, -1):
+        readings.append(
+            MeterReading(site_id=1,
+                         temp_c=i * 1.0,
+                         wh_used=i * 1.0,
+                         wh_generated=i * 1.0,
+                         timestamp=time))
+        time = time + datetime.timedelta(minutes=1)
+    yield readings
+
+
+def _test_insert_and_retrieve(readings: List[MeterReading],
+                              metric_dao: MetricDaoRedisTimeseries, limit: int):
+    for reading in readings:
+        metric_dao.insert(reading)
+
+    measurements = metric_dao.get_recent(TESTING_SITE_ID, MetricUnit.WH_GENERATED, NOW,
+                                         limit)
+    assert len(measurements) == limit
+
+    i = limit
+    for measurement in measurements:
+        assert measurement.value == i * 1.0
```

Comment:
The sorted set DAO uses `(i - 1) * 1.0` here. But that's not the set of values that this DAO returns... so which is right? The Java DAO that this is based on exhibits the same behavior.